{
    "DWF": {
        "references": [
            "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2021-30500",
            "https://bugzilla.redhat.com/show_bug.cgi?id=1948692",
            "https://www.whitesourcesoftware.com/vulnerability-database/CVE-2021-30500",
            "https://security.archlinux.org/CVE-2021-30500",
            "https://www.suse.com/security/cve/CVE-2021-30500/"
        ]
    },
    "MITRE": {
        "CVE_data_meta": {
            "ASSIGNER": "secalert@redhat.com",
            "ID": "CVE-2021-30500",
            "STATE": "PUBLIC"
        },
        "affects": {
            "vendor": {
                "vendor_data": [
                    {
                        "product": {
                            "product_data": [
                                {
                                    "product_name": "upx",
                                    "version": {
                                        "version_data": [
                                            {
                                                "version_value": "upx 4.0"
                                            }
                                        ]
                                    }
                                }
                            ]
                        },
                        "vendor_name": "n/a"
                    }
                ]
            }
        },
        "data_format": "MITRE",
        "data_type": "CVE",
        "data_version": "4.0",
        "description": {
            "description_data": [
                {
                    "lang": "eng",
                    "value": "Null pointer dereference was found in upx PackLinuxElf::canUnpack() in p_lx_elf.cpp,in version UPX 4.0.0. That allow attackers to execute arbitrary code and cause a denial of service via a crafted file."
                }
            ]
        },
        "problemtype": {
            "problemtype_data": [
                {
                    "description": [
                        {
                            "lang": "eng",
                            "value": "CWE-476"
                        }
                    ]
                }
            ]
        },
        "references": {
            "reference_data": [
                {
                    "name": "https://bugzilla.redhat.com/show_bug.cgi?id=1948692",
                    "refsource": "MISC",
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1948692"
                },
                {
                    "name": "https://github.com/upx/upx/issues/485",
                    "refsource": "MISC",
                    "url": "https://github.com/upx/upx/issues/485"
                },
                {
                    "name": "https://github.com/upx/upx/commit/90279abdfcd235172eab99651043051188938dcc",
                    "refsource": "MISC",
                    "url": "https://github.com/upx/upx/commit/90279abdfcd235172eab99651043051188938dcc"
                }
            ]
        }
    },
    "NVD": {
        "configurations": {
            "CVE_data_version": "4.0",
            "nodes": []
        },
        "cve": {
            "CVE_data_meta": {
                "ASSIGNER": "secalert@redhat.com",
                "ID": "CVE-2021-30500"
            },
            "data_format": "MITRE",
            "data_type": "CVE",
            "data_version": "4.0",
            "description": {
                "description_data": [
                    {
                        "lang": "en",
                        "value": "Null pointer dereference was found in upx PackLinuxElf::canUnpack() in p_lx_elf.cpp,in version UPX 4.0.0. That allow attackers to execute arbitrary code and cause a denial of service via a crafted file."
                    }
                ]
            },
            "problemtype": {
                "problemtype_data": [
                    {
                        "description": []
                    }
                ]
            },
            "references": {
                "reference_data": [
                    {
                        "name": "https://github.com/upx/upx/issues/485",
                        "refsource": "MISC",
                        "tags": [],
                        "url": "https://github.com/upx/upx/issues/485"
                    },
                    {
                        "name": "https://github.com/upx/upx/commit/90279abdfcd235172eab99651043051188938dcc",
                        "refsource": "MISC",
                        "tags": [],
                        "url": "https://github.com/upx/upx/commit/90279abdfcd235172eab99651043051188938dcc"
                    },
                    {
                        "name": "https://bugzilla.redhat.com/show_bug.cgi?id=1948692",
                        "refsource": "MISC",
                        "tags": [],
                        "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1948692"
                    }
                ]
            }
        },
        "impact": {},
        "lastModifiedDate": "2021-05-27T00:15Z",
        "publishedDate": "2021-05-27T00:15Z"
    }
}